app/controllers/employees_controller.rb

```diff
@@ -16,6 +17,11 @@ def new
     @branches = Branch.all
   end
 
+  #Test 
+  def test 
+    layout "test"
+  end 
+
```

Comment:
what is this test for?

Reply:
We were just trying some stuff. now doesn´t exist. Thank you.